Ion.egg-info/requires.txt

```diff
@@ -1,105 +1,95 @@
 alabaster==0.7.10
-argon2-cffi==16.3.0
-asgi-redis==1.4.3
-asgiref==1.1.2
-asn1crypto==0.23.0
-attrs==17.2.0
-autobahn==17.9.3
-Automat==0.6.0
-Babel==2.5.1
-bcrypt==3.1.3
+asgiref==2.3.0
+asn1crypto==0.24.0
+async-timeout==2.0.1
+attrs==17.4.0
+autobahn==18.4.1
+automat==0.6.0
+babel==2.5.3
+backcall==0.1.0
+bcrypt==3.1.4
 beautifulsoup4==4.6.0
-bleach==2.0.0
-certifi==2017.7.27.1
-cffi==1.11.0
-channels==1.1.6
+bleach==2.1.3
+certifi==2018.4.16
+cffi==1.11.5
+channels==2.1.1
```

Comment:
This requires some rewriting of the code (as I'm sure is the case with some other major version updates). See #554

Reply:
Locked the versions of all top-level deps, PTAL.